GoodFirstIssue-ToDoApp/Screens/ItemEdit/ItemEditViewController.swift

```diff
@@ -7,19 +7,16 @@
 //
 
 import UIKit
+import RealmSwift
 
 class ItemEditViewController: UIViewController, ItemEditViewProtocol {
+    private var realm: Realm?
```

Comment:
以下のようにすれば各所で!しなくても良いのではないかと思います
```suggestion
    private let realm: Realm = try! Realm()
```

Reply:
var itemArray: Results<Item> = try! Realm().objects(Item.self)
という書き方にまとめてみました。
ありがとうございます。